tests/smoke/recommenders/recommender/test_newsrec_model.py

```diff
@@ -115,7 +115,7 @@ def test_model_naml(mind_resource_path):
     assert model.run_eval(valid_news_file, valid_behaviors_file) is not None
     assert isinstance(
         model.fit(
-            train_news_file, train_behaviors_file, valid_news_file, valid_behaviors_file
+            train_news_file, train_behaviors_file, valid_news_file, valid_behaviors_file,step_limit=1
```

Comment:
If 10 steps is less than a second, can we use that instead of 1?

Reply:
ofcourse, we can use 10 instead of 1,but as per the output step_limit = 7 is still more preferrable right.

Reply:
I wouldn't take the times too strictly because they could change in other computers. Feel free to choose a number that is round (5, 10, 20) and that doesn't take to long

Reply:
okay